### ISSUE TITLE
Possible converters from fbx to a particular format

### ISSUE THREAD

**Issue by ghost** [pdsharma0/bsp2fbx#1]
anyway of converting the fbx to .map, .rmf, .vmf and obj etc?

just wondering if that's possible.

**Comment by pdsharma0** [comment]
I am sure it's possible if those file formats (MAP, RMF, VMF) are well described somewhere. 

I am just trying to understand your use-case here though, do you want to modify the fbx generated by this utility and then convert them again to goldsrc engine files to be used in game?

**Comment by ghost** [comment]
yeah essentially what i want to do with them both source and goldsrc

**Comment by pdsharma0** [comment]
Ok, It makes sense to have both utilities to be able to make this useful. Do you have any resources where I can learn more about these formats? 

I am still working on getting the goldsrc format to FBX done first, the source will follow later.
